paymentsheet/src/main/java/com/stripe/android/paymentsheet/specifications/Spec.kt

```diff
@@ -88,7 +88,11 @@ sealed class SectionFieldSpec(open val identifier: IdentifierSpec) {
     data class Country(val onlyShowCountryCodes: Set<String> = emptySet()) :
         SectionFieldSpec(IdentifierSpec("country"))
 
-    object IdealBank : SectionFieldSpec(IdentifierSpec("bank"))
+    data class SimpleDropdown(
```

Comment:
Seems this will be used for Bank types only, as it has a `bankType` property, if so can we rename it to bank specific like `SimpleBankDropDown`?